rmf_robot_sim_gz_plugins/src/TeleportDispenser.cpp

```diff
@@ -37,12 +37,13 @@
 
 #include <rclcpp/rclcpp.hpp>
 #include <rmf_fleet_msgs/msg/fleet_state.hpp>
+#include <rmf_dispenser_msgs/msg/dispenser_state.hpp>
+#include <rmf_dispenser_msgs/msg/dispenser_result.hpp>
+#include <rmf_dispenser_msgs/msg/dispenser_request.hpp>
 
-#include <rmf_robot_sim_common/dispenser_common.hpp>
 #include <rmf_robot_sim_common/utils.hpp>
```

Comment:
Is it possible to also get rid of this and fully remove the dependency? Removing `SimEntity` should help I believe